### PR TITLE
Versioning docs and examples for release-1.3.

### PR DESCRIPTION
**Pull request by david-mcmahon**
Still not looking right.  Better, but not right. 
Ref https://github.com/kubernetes/release/pull/14

### CODE REVIEW

File: docs/admin/kube-apiserver.md [hunk 1]
```diff
@@ -18,6 +18,11 @@
 If you are using a released version of Kubernetes, you should
 refer to the docs that go with that version.
 
+<!-- TAG RELEASE_LINK, added by the munger automatically -->
+<strong>
+The latest release of this document can be found
+[here](http://releases.k8s.io/release-1.2/docs/admin/kube-apiserver.md).
```
Comment (lavalamp) [inline docs/admin/kube-apiserver.md]: we should not link to release-1.2 from release 1.3?

File: docs/README.md [hunk 2]
```diff
@@ -3,7 +3,7 @@
 
 <!-- END MUNGE: UNVERSIONED_WARNING -->
 
-# Kubernetes Documentation: releases.k8s.io/v1.3.0-beta.0
+# Kubernetes Documentation: releases.k8s.io/release-1.3
```
Comment (lavalamp) [inline docs/README.md]: Is this supposed to be a version or a branch?

**Comment by david-mcmahon** [comment]
This repaired all of those locations where the release version was in place of the release branch, but there are still release-1.2 placeholders being put in.  At least hack/verify-munge-docs.sh is now happy,

File: docs/README.md (hunk 2 above)
Comment (david-mcmahon) [inline docs/README.md]: Supposed to be a branch.  See also https://github.com/kubernetes/release/pull/14

File: docs/admin/kube-apiserver.md (hunk 1 above)
Comment (david-mcmahon) [inline docs/admin/kube-apiserver.md]: versionize-docs.sh runs update-generated-docs.sh which runs update-munge.docs.sh which is going to then insert these RELEASE_LINKs into the docs.

**Comment by david-mcmahon** [comment]
PTAL and merge if OK.

**Comment by lavalamp** [comment]
This looks correct now, thanks.

**Comment by lavalamp** [comment]
LGTM

**Pull request merged**